src/exprs.c

```diff
@@ -1458,8 +1460,18 @@ void            PrintUnknownExpr (
 **  This sometimes puts in superfluous parenthesis: 2 * f( (3 + 4) ), since it
 **  doesn't know that a function call adds automatically parenthesis.
 */
-UInt            PrintPrecedence;
+UInt PrintPrecedence;
+
+};
+
+static ModuleStateOffset ExprsStateOffset = -1;
 
+extern inline struct ExprsState * ExprsState(void)
```

Comment:
extern inline with function definition?

Reply:
Yes. The extern ensures that the compiler emits an actual instance of the function; that's useful so that one can invoke it from a debugger.

Reply:
I see, thanks for the clarification.